### ISSUE TITLE
CBR matrix not always correctly refreshing when first connection established

### ISSUE THREAD

**Issue by christophe-calmejane** [christophe-calmejane/Hive#89]
For some devices, the first (one or more) connection established do not show in the matrix (CBR mode only). It's only after a few other channel connections that the previous are visible.
Reloading the controller correctly displays the connection, meaning it's a refresh issue.

**Comment by christophe-calmejane** [comment]
A `if condition` in `channelConnectionManager.cpp` is incorrect.

```c++
else if (clusterIndex + baseCluster == mapping.clusterOffset && clusterChannel == mapping.clusterChannel && mapping.streamIndex == streamIndex)
```
should be

```c++
else if (clusterIndex - baseCluster == mapping.clusterOffset && clusterChannel == mapping.clusterChannel && mapping.streamIndex == streamIndex)
```

**Comment by christophe-calmejane** [comment]
Fixed in ddafdb90ddf93aa521f651b372d69b79a869493d
